lib/matplotlib/quiver.py

```diff
@@ -569,6 +628,19 @@ def set_UVC(self, U, V, C=None):
             self.set_array(C)
         self.stale = True
 
+    def set_offsets(self, xy):
```

Comment:
change to `set_XY` to match input naming.

Reply:
Good point. I added `set_offsets` to be consistent with other collections. An alternative would be to rename `XY` to `offsets` everywhere? Maybe for another PR! 😅

Reply:
I have keep both `set_XY` and `set_offsets`, with `set_XY` being an alias of `set_offsets` and documented as such.

I still think that it would be simple to keep only `set_offsets` because of its consistency with others collections but also because it is possible to use `set_X` and `set_Y` which match the constructor (`set_XY` doesn't match anything really).

Reply:
I removed `set_XY`/`get_XY` to keep the API simple for the following reason:
- it doesn't match the constructor arguments
- the `X`, `Y` setter/getter are matching the constructor arguments
- it is an alias to setter/getter of the base class (`Collection`).